File: money-local-service/src/main/java/com/hmpay/moneylocal/application/service/RechargeMoney.java
package com.hmpay.moneylocal.application.service;


import com.hmpay.common.UseCase;
import com.hmpay.moneylocal.application.port.in.RechargeMoneyCommand;
import com.hmpay.moneylocal.application.port.in.RechargeMoneyUseCase;
import jakarta.transaction.Transactional;
import lombok.RequiredArgsConstructor;



@RequiredArgsConstructor
@UseCase
@Transactional
public class RechargeMoney implements RechargeMoneyUseCase {


	@Override
	public void rechargeMoney(RechargeMoneyCommand command) {

	}
}

File: money-local-service/src/main/kotlin/com/hmpay/moneylocal/application/service/RechargeMoney.kt
package com.hmpay.moneylocal.application.service

import com.hmpay.common.UseCase
import com.hmpay.moneylocal.application.port.`in`.RechargeMoneyCommand
import com.hmpay.moneylocal.application.port.`in`.RechargeMoneyUseCase
import jakarta.transaction.Transactional

@UseCase
@Transactional
class RechargeMoney : RechargeMoneyUseCase {
    override fun rechargeMoney(command: RechargeMoneyCommand) {
        // TODO: implement recharge logic for money-local-service
    }
}
